File: PAT/BLevel/1010/1010.cpp
// 1010 一元多项式求导
// auguralpha
// 2019-02-11
#include <cstdio>

int main() {
    int a, b;
    bool mark = false;
    while(scanf("%d%d", &a, &b) != EOF) {
        if(b != 0) {
            printf("%s%d %d", (mark ? " " : ""), a * b, b - 1);
            mark = true;
        }
    }
    if(!mark) {
        puts("0 0");
    }
    else {
        puts("");
    }
    return 0;
}

File: PAT/BLevel/1010/1010.go
// 1010 一元多项式求导
// auguralpha
// 2020-03-25

package main

import "fmt"

func main() {
	var a, b int
	var mark bool = false
	for true {
		_, err := fmt.Scanf("%d%d", &a, &b)
		if err != nil {
			break
		}

		if b != 0 {
			if mark {
				fmt.Printf(" %d %d", a * b, b - 1)
			} else {
				fmt.Printf("%d %d", a * b, b - 1)
				mark = true
			}
		}
	}

	if !mark {
		fmt.Println("0 0")
	} else {
		fmt.Println("")
	}
}
